### PR TITLE
Initial codebase

### PR DESCRIPTION
**Pull request by Runi-VN**
# Initial codebase
image

- Overhauled .travis.yml, make sure to give it a look.
- Removed all unused stuff: Database, utility and the like.
- Added a simple facade + testing
- Added a few REST-endpoints (taking use of the facade) + testing

## Stuff to consider
- ~~It currently deploys to https://runivn.dk/3SemProject_BackEnd/~~
    - ~~Maybe we should name it something else..? Suggestions?~~ 
changed to https://runivn.dk/3SEMPROJECT/
- The DTO is empty and unused.
- I had to manually add CORS and errorhandling to ApplicationConfig. Keep this in mind.

## Stuff we still need to change:
- [x] Remove unused dependencies
- [x] Remove unused **test** dependencies
  - There are still a few left but I think they are dependant on eachother.

### CODE REVIEW

**Comment by Runi-VN** [comment]
It works 🎉 
image
Will try to trigger re-deployment with something new. (to see if Tomcat actually deploys, unlike in CA1)

**Comment by Runi-VN** [comment]
Yup, deployment works continually! 🎉 
image

File: pom.xml [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+            <groupId>org.eclipse.persistence</groupId>
+            <artifactId>org.eclipse.persistence.jpa.modelgen.processor</artifactId>
+            <version>2.5.2</version>
+            <scope>provided</scope>
+        </dependency>
+        <dependency>
+            <groupId>mysql</groupId>
+            <artifactId>mysql-connector-java</artifactId>
+            <version>8.0.17</version>
+        </dependency>
```
Comment (MrHardcode) [inline pom.xml]: My guess is that we can remove all these dependencies since we don't plan on using entities and a database

**Comment by MrHardcode** [comment]
Looks good

**Review by MrHardcode — APPROVED** [review]

**Review by MalteMagnussen — APPROVED** [review]

**Pull request merged**